server.js

```diff
@@ -1,42 +1,39 @@
 /*eslint-env node*/
-"use strict";
-
-const fs = require("fs");
-const path = require("path");
-const { performance } = require("perf_hooks");
-const request = require("request");
-const { URL } = require("url");
-
-const chokidar = require("chokidar");
-const compression = require("compression");
-const express = require("express");
-const mkdirp = require("mkdirp");
-const rimraf = require("rimraf");
-const yargs = require("yargs").options({
-  port: {
-    default: 8080,
-    description: "Port to listen on.",
-  },
-  public: {
-    type: "boolean",
-    description: "Run a public server that listens on all interfaces.",
-  },
-  "upstream-proxy": {
-    description:
-      'A standard proxy server that will be used to retrieve data.  Specify a URL including port, e.g. "http://proxy:8000".',
-  },
-  "bypass-upstream-proxy-hosts": {
-    description:
-      'A comma separated list of hosts that will bypass the specified upstream_proxy, e.g. "lanhost1,lanhost2"',
-  },
-  help: {
-    alias: "h",
-    type: "boolean",
-    description: "Show this help.",
-  },
-});
-
-const {
+import fs from "fs";
+import path from "path";
+import { performance } from "perf_hooks";
+import request from "request";
+import { URL } from "url";
+
+import chokidar from "chokidar";
+import compression from "compression";
+import express from "express";
+import mkdirp from "mkdirp";
+import rimraf from "rimraf";
+import yargs from "yargs";
+
+const argv = yargs(process.argv)
+  .options({
+    port: {
```

Comment:
On line 482, the error message for an incorrect port mentions `server.cjs` instead of `server.js`.